### PR TITLE
Get Involved Page

### LINKED ISSUE #53: UI Design for Get Involved page
**Here is the final design. Please refer to screenshot below.**

get_involved_final

**You can refer to the following image for further details on creating the page :**

get_involved_final_documentation

You can refer to the following image for further details on creating the page :
https://drive.google.com/drive/folders/1pk5qIzxfIfYJriUQ0JkiXxDWLdVQJ3cB

You can get the relevant text from the present website itself : https://evalai.cloudcv.org/get-involved

Please comment below if you want to work on this or in case of any question/suggestion.

Mentors for EvalAI-ngx Project (@aka-jain @spyshiv @shivaniprakash95 @RishabhJain2018 @deshraj) will try to respond to this thread ASAP so that you are not blocked.

### PR DESCRIPTION
**Pull request by live-wire**
Changes for Get Involved Page available at path `/get-involved` 
Supposed to close #53 

> Header will look better once #61 is merged

### CODE REVIEW

File: src/app/get-involved/get-involved.component.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<app-header-static></app-header-static>
+<div class="container-top container align-center get-involved-container"> 
+  <div class="get-involved-title">
+  	Get Involved
+  </div>
+  <div class="get-involved-description">
+  	{{description}}
+  </div>
+  <div class="get-involved-section">
```
Comment (aka-jain) [inline src/app/get-involved/get-involved.component.html]: You forgot to add row here. 
Note: If you use `col` then please wrap it in a `row`. @live-wire

File: src/app/get-involved/get-involved.component.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  	</div>
+  </div>
+  <hr>
+
+  <div class="get-involved-section">
+  	<div class="col-lg-6 col-md-6 col-sm-12 col-xs-12">
+  		<div class="get-involved-section-title left">Press</div>
+  		<div class="get-involved-section-body left">If you're a journalist who would like to help spread the word about EvalAI or CloudCV, please email <a href="mailto:admin@cloudcv.org" class="red-link">admin@cloudcv.org</a>.</div>
+  	</div>
+  	<div class="col-lg-6 col-md-6 col-sm-12 col-xs-12">
```
Comment (aka-jain) [inline src/app/get-involved/get-involved.component.html]: Please fix row issue for all the col. @live-wire

**Review by aka-jain — COMMENTED** [review]
@live-wire please use px for margin and paddings instead of `%`, it might create problem for different screens.

**Comment by live-wire** [comment]
Hey @aka-jain, I updated this PR with the suggested changes. Please have a look. 🍕

**Comment by live-wire** [comment]
jun-4-2018-10_49-pm
☝️ This is how the `/get-involved` page looks like.

Please have a look @spyshiv @aka-jain @RishabhJain2018 🥔 

cc: @shivaniprakash95

**Comment by RishabhJain2018** [comment]
LGTM :)

**Comment by aka-jain** [comment]
Resolve conflicts please @live-wire

**Review by aka-jain — APPROVED** [review]

**Comment by live-wire** [comment]
Done @aka :+1:

**Pull request merged**